### ISSUE TITLE
Problem with fast forward / rewind

### ISSUE THREAD

**Issue by jeffkontak** [SourMesen/Mesen#582]
Hello! First let me say I'm blown away by your emulator. I've used various NES emulators for years and your UI and display capabilities are flawless in comparison. Here is my issue. I use a newly purchased easySMX wired controller. Mesen automatically confured it when I first opened the software. All was seamless. Then I noticed that it assigned fast forward and rewind functions to the L2 / R2 triggers. I wanted to disable that because I don't like ever using it and accidently hitting them. I went into the input options menu, clicked setup, cleared all the inputs, and then did not assign those two. I expected them to be disabled but the same two triggers continue to rewind and fast forward. I've put those functions on keys. I've deleted info for a second controller that was presaved. I've unchecked the automatically configure controllers box. Nothing fixes it. Please let me know how I can disable those functions from my controller if I'm missing something.

**Comment by SourMesen** [comment]
Thanks, happy to hear you're liking it!

From your description, it sounds like you might not have looked in the right place to disable those shortcuts.  The input options are only for the console's controllers & their buttons, all other shortcuts are in Preferences -> Shortcut Keys:
![image](https://user-images.githubusercontent.com/21093973/53138243-a4813680-3553-11e9-820f-0a92d8adb0b9.png)

Try clearing the rewind/fast forward keys here (you can right-click on them to clear them) and see if it helps, if you haven't done so already.

**Comment by jeffkontak** [comment]
Thank you for the swift reply. I saw you on the forums on romhacking.net also and it’s great to see you are so invested in your work. I run some dragon quest projects there myself as Chicken Knife.

Anyway, your suggestion totally fixed the issue. My feedback is that I guess I found it unintuitive that your emulator would have automatically assigned functions like that to controller buttons. Normally all the functions assigned to a controller are consolidated on the input settings menu. 

That being said, I can’t think you enough for your work. It literally blows me away. I spent a whole day recently bragging to all my gamer friends about mesen after discovering how good it is. 😄

Take care!
